code/game/objects/items/storage/boxes.dm

```diff
@@ -1174,6 +1174,18 @@
 		/obj/item/stock_parts/matter_bin/bluespace = 3)
 	generate_items_inside(items_inside,src)
 
+/obj/item/storage/box/stockparts/cybersun
+	name = "box of emergency stock parts"
+	desc = "Contains all stock parts you might need to setup basic R&D."
+	icon_state = "syndiebox"
+
+/obj/item/storage/box/stockparts/cybersun/PopulateContents()
+	var/static/items_inside = list(
+		/obj/item/stock_parts/matter_bin/bluespace = 6,
+		/obj/item/stock_parts/manipulator/femto = 5,
+		/obj/item/reagent_containers/glass/beaker = 4)

```

Comment:
what's with the T4 parts?

Reply:
It's using T4 parts right now, I just reduced the amount of them.

Reply:
WHAT IS THIS RUIN 😠

Reply:
Alright alright, it always had Deluxe stock parts boxes, same with lavaland base.
I'm okay with lowering the tier of parts, but not to the first tier.

Reply:
make it first tier

Reply:
@Qustinnus 
No.

Reply:
then ill open a pr myself 😏

Reply:
I may see a reason to lower the tier by a bit, but I don't see any reasons to set it to the lowest one.